test/source-updater.test.js

```diff
@@ -1,232 +1,769 @@
-import SourceUpdater from '../src/source-updater';
+import document from 'global/document';
+import window from 'global/window';
 import QUnit from 'qunit';
 import videojs from 'video.js';
-import { useFakeMediaSource } from './test-helpers';
+import SourceUpdater from '../src/source-updater';
+import { mp4Video, mp4Audio } from './test-segments';
 
 QUnit.module('Source Updater', {
   beforeEach() {
```

Comment:
seems like this should be a `before` instead(and we need a `after` to clean up)? or we need a `afterEach` to clean up the media source and video element created before each test

Reply:
Added an `afterEach` to clean up the media source. It's a `beforeEach` instead of a `before` so we have a clean instance of source updater (and source buffers) on each test.

Reply:
has this been pushed up already?

Reply:
Yup. https://github.com/videojs/http-streaming/pull/272/commits/526bc0931705b6d433531c4fb0a2d588c700a548

Reply:
Might want to also clean up `this.mediaSource` in the `afterEach` to make sure the old ones die